executor/aggfuncs/func_varpop.go

```diff
@@ -0,0 +1,66 @@
+package aggfuncs
+
+import (
+	"github.com/pingcap/errors"
+	"github.com/pingcap/tidb/sessionctx"
+	"github.com/pingcap/tidb/util/chunk"
+)
+
+type varPop4Float64 struct {
+	baseAggFunc
+}
+
+type partialResult4Float64 struct {
```

Comment:
It's better to use the algorithm described by Chan, Golub, and LeVeque in "Algorithms for computing the sample variance: analysis and recommendations" to calculate the variance. You can find how apache hive implement this at here: https://github.com/apache/hive/blob/master/ql/src/java/org/apache/hadoop/hive/ql/udf/generic/GenericUDAFVariance.java#L194

Reply:
Okay. I will refer to it.